### PR TITLE
chore: depcheck and missing deps - doorway uic

### PR DESCRIPTION
**Pull request by emilyjablonski**
## Description

No explicit issue - I ran [depcheck](https://www.npmjs.com/package/depcheck) in doorway-ui-components and removed unused packages. ListingMap was also not being consumed so I removed that as well to remove the map dependency.

## How Can This Be Tested/Reviewed?

I poked around to ensure nothing significant broke and that the map still works as expected.

## Author Checklist:

- [ ] Added QA notes to the issue with applicable URLs
- [ ] Reviewed in a desktop view
- [ ] Reviewed in a mobile view
- [ ] Reviewed considering accessibility
- [ ] Added tests covering the changes
- [ ] Made corresponding changes to the documentation
- [ ] Ran `yarn generate:client` and/or created a migration when required

## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates

### CODE REVIEW

File: doorway-ui-components/.storybook/main.js [hunk 1]
```diff
@@ -11,18 +11,6 @@ module.exports = {
     "@storybook/addon-a11y",
     "@storybook/addon-knobs",
     "@geometricpanda/storybook-addon-badges",
-    {
-      /**
-       * Fix Storybook issue with PostCSS@8
-       * @see https://github.com/storybookjs/storybook/issues/12668#issuecomment-773958085
-       */
-      name: "@storybook/addon-postcss",
```
Comment (emilyjablonski) [inline doorway-ui-components/.storybook/main.js:19]: Removed this addon, it wasn't needed + was causing a security vulnerability

File: doorway-ui-components/index.ts [hunk 2]
```diff
@@ -112,7 +112,6 @@ export * from "./src/page_components/listing/ContentAccordion"
 export * from "./src/page_components/listing/ListingCard"
 export * from "./src/page_components/listing/ListingDetailHeader"
 export * from "./src/page_components/listing/ListingDetails"
-export * from "./src/page_components/listing/ListingMap"
```
Comment (emilyjablonski) [inline doorway-ui-components/index.ts:115]: We weren't using this anywhere

**Comment by jaredcwhite** [comment]
@emilyjablonski Nice! Were you ready for a review/approval on this?

**Comment by emilyjablonski** [comment]
@jaredcwhite should be ready now!

**Review by jaredcwhite — APPROVED** [review]
LGTM

**Pull request merged**